Core/src/org/sleuthkit/autopsy/contentviewers/MessageContentViewer.java

```diff
@@ -633,16 +660,20 @@ static private String cleanseHTML(String htmlInString) {
         return doc.html();
     }
 
-    private static class AttachmentsChildren extends Children.Keys<AbstractFile> {
+    
+    /**
+     * Creates child nodes for message attachments. 
+     */
+    final class AttachmentsChildren extends Children.Keys<Attachment> {
```

Comment:
It's not clear to me why this class was changed to be non-static and to have package access instead of private access. Is this a necessary change?